src/Model.hs

```diff
@@ -92,15 +94,14 @@ update
 update = do
   c       <- asks ccCfg
   dateNow <- liftIO $ fmap (view _zonedTimeToLocalTime) getZonedTime
-  let (textday, date) =
+  let (textday, d) =
```

Comment:
I don't think there is a warning for too long variable names ... Hmmm :)

Reply:
Sadly there isn't. :( 
On a more serious note, GHC gave warnings for `date` shadowing some function from Thyme so I just renamed `date`.